packages/spacecat-shared-data-access/src/models/importer/import-job.js

```diff
@@ -21,148 +21,150 @@ export const ImportJobStatus = {
   FAILED: 'FAILED',
 };
 
+export const ImportOptions = {
+  ENABLE_JAVASCRIPT: 'enableJavascript',
```

Comment:
We check for `pageLoadTimeout` in the content-scraper : https://github.com/adobe/spacecat-content-scraper/blob/main/src/handlers/abstract-handler.js#L270. Is that something we could like to add in as well?

Reply:
Is the pageLoadTimeout a user provided property or one of our default properties that is automatically added by us?

Reply:
It has a default value if not provided by the user. But, its value could be provided by the user - based on the current helix-importer-ui tool.